ci/_build_functions.sh

```diff
@@ -35,14 +35,16 @@ function push_helm_chart() {
 
   set -ex
   # this loop is to allow for concurent builds: https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/1853 
-  for i in 1..${retry_count}; do
+  for i in $(seq 0 $((retry_count-1)))
```

Comment:
Why does this fix the issue?

Reply:
The retry doesn't work at the moment
https://github.com/SumoLogic/sumologic-kubernetes-collection/actions/runs/5186860707/jobs/9348502256
wanted to fix it to see if it makes a difference

Reply:
Do you know why the current state doesn't work?

Reply:
yeah, it looks like an issue with the retry condition
```
elif [[ "${i}" == "${retry_count}" ]]
```
https://github.com/SumoLogic/sumologic-kubernetes-collection/actions/runs/5186860707/jobs/9348502256#step:3:295